dex-load/src/main/scala/com/wavesplatform/dex/load/TankGenerator.scala

```diff
@@ -261,34 +283,37 @@ object TankGenerator {
 
   def placeOrdersForCancel(accounts: List[PrivateKeyAccount], requestsCount: Int, pairsFile: Option[File]): Unit = {
     println("Placing some orders to prepare cancel-order requests... ")
+
     val pairs = mkPairsAndDistribute(accounts, pairsFile)
 
-    for (i <- 0 to requestsCount) {
-      try {
-        val acc  = accounts(new Random().nextInt(accounts.length - 1))
-        val pair = pairs(new Random().nextInt(pairs.length - 1))
-        services.matcher.placeOrder(
-          acc,
-          settings.matcherPublicKey,
-          pair,
-          Type.BUY,
-          settings.defaults.minimalOrderPrice,
-          settings.defaults.minimalOrderAmount,
-          System.currentTimeMillis + 60 * 60 * 24 * 20 * 1000,
-          settings.defaults.matcherFee,
-          null,
-          false
-        )
-        if (i % 50 == 0) println(s"$i orders of $requestsCount have been placed")
-      } catch {
-        case e: Exception => println(e.getMessage)
+    val futures = (0 to requestsCount).map(_ => {
+      Future {
+        blocking {
```

Comment:
`blocking` makes things worse in this case: it leads to additional thread creation, so the server will face more than 5 simultaneous requests and probably will fail to process some of requests. Also see our chat at 11:16 AM (yesterday).

Suggest to remove `blocking` wrapper.

Reply:
fixed